src/settings/LoanHistory/utils/normalize.js

```diff
@@ -0,0 +1,48 @@
+import {
+  cloneDeep,
+  forEach,
+  set,
+  isArray,
+  isString,
+} from 'lodash';
+import { change } from 'redux-form';
+
+import { closingTypesMap } from '../../../constants';
+
+const checkInterval = ({ data, dispatch }) => {
+  const newData = cloneDeep(data);
+
+  forEach(Object.entries(data.closingType), ([key, value]) => {
```

Comment:
```suggestion
  forEach(data.closingType, (key, value) => {
```
Should do the same thing

Reply:
More exactly
```
forEach(data.closingType, (value, key) => {
```
Anyway thanks a lot for the hint.